src/VirtualClient/VirtualClient.Main/Program.cs

```diff
@@ -192,6 +192,9 @@ internal static CommandLineBuilder SetupCommandLine(string[] args, CancellationT
                 // --contentStore
                 OptionFactory.CreateContentStoreOption(required: false),
 
+                // --contentPathPattern
+                OptionFactory.CreateContentPathPatternOption(required: false, "{experimentId}/{agentId}/{toolName}/{role}/{scenario}"),
```

Comment:
I would rather this default pattern set later in the code, not all the way front in Program.

Reply:
Modified as suggested. Now the default is set in VCComponent